modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/java/com/liferay/portal/search/tuning/rankings/web/internal/portlet/action/ViewResultRankingsMVCRenderCommand.java

```diff
@@ -89,4 +91,7 @@ public String render(
 	@Reference
 	protected Sorts sorts;
 
+	@Reference
```

Comment:
Im thinking that this needs to be an "optional" reference for when LES is disabled and there is no available SXPBlueprintTitleProvider (i think core infra now handles this by using snapshot.get(), see Snapshot<SyncReindexManager>)

did you test to see if the rankings UI still worked with LES disabled?